frontend/src/Components/Header/Header.tsx

```diff
@@ -1,38 +1,34 @@
 import React, { useState } from "react";
-import { Container, Wrapper } from "./HeaderStyle";
-import { useMediaQuery } from 'react-responsive'
+import { Container, Wrapper } from "./Header.style";
+import { useMediaQuery } from "react-responsive";
 import { Link } from "react-router-dom";
-import { Navbar } from "../draft/NavBar/Navbar";
 
-type HeaderProps ={
-  headerOf: string,
-}
-const Header: React.FC<HeaderProps> = ({headerOf}) => {
+type HeaderProps = {
+  headerOf: string;
+};
+const Header: React.FC<HeaderProps> = ({ headerOf }) => {
   const [toggleMenu, setToggleMenu] = useState(false);
-  const [header, setHeader] = useState(headerOf)
+  const [header, setHeader] = useState(headerOf);
```

Comment:
> const [header, setHeader] = useState(headerOf);

Why do you need to declare this variable after already receiving it from the props?
You can just use "headerOf" directly

Reply:
right! 
Thanks!